com/uber/kafkaSpraynozzle/KafkaLog.java

```diff
@@ -64,6 +68,11 @@ public void run() {
                 }
             }
             System.out.println(String.format(LOG_FORMAT, this.topic, enqueued, filteredOut, clogged, this.url, posting, postSuccess, postFailure));
+            statsReporter.count("enqueued", enqueued);
+            statsReporter.count("paused", clogged);
+            statsReporter.count("posted", posting);
+            statsReporter.count("postSuccess", postSuccess);
+            statsReporter.count("postFailure", postFailure);
```

Comment:
i opt'd to put the stats counting here, instead of throughout the code.  This keeps all "monitoring" isolated to 1 location and keeps it out of the way from the regular code flow.  I'm thought about a few other options too, and am flexible to how we want this to work:
1) reverting KafkaLog and putting stats logging alongside where we enque events
2) put an interface over KafkaLog and have it be swappable.  So we can use StatsD or KafkaLog or any other monitoring class.  I kind of like having both the logging and stats counting, but that might just be me.